examples/inverse/plot_use_sourcemorph.py

```diff
@@ -0,0 +1,250 @@
+    # orientation is optimized by setting pick_ori to 'max-power'.
+    # weight_norm can also be set to 'unit-noise-gain'. Source orientation can
+    # also be 'normal' (but only when using a surface-based source space) or
+    # None, which computes a vector beamfomer. Note, however, that not all
+    # combinations of orientation selection and weight normalization are
+    # implemented yet.
+    filters = make_lcmv(evoked.info, forward, data_cov, reg=0.05,
+                        noise_cov=noise_cov, pick_ori='max-power',
+                        weight_norm='nai')
+
+    # Apply this spatial filter to the evoked data.
+    stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+    # take absolute values for plotting
+    stc.data[:, :] = np.abs(stc.data)
+
+    # Save result in stc files if desired
+
+    # select time window (tmin, tmax) in ms - consider changing for real data
+    # scenario, since those values were chosen to optimize computation time
+    stc.crop(0.087, 0.087)
+
+    src = forward['src']
+
+    if fname is not None:
+        stc.save(fname)
+
+    return stc, src
+
+
+# Setup path
+data_path = mne.datasets.sample.data_path()
+t1_sample = data_path + '/subjects/sample/mri/brain.mgz'
+t1_fsaverage = data_path + '/subjects/fsaverage/mri/brain.mgz'
+fname_fwd = data_path + '/MEG/sample/sample_audvis-meg-vol-7-fwd.fif'
+subjects_dir = data_path + '/subjects'
+
+# read precomputed data
+compute_lcmv_example_data_surf(data_path, fname='lcmv-surf')
+compute_lcmv_example_data_vol(data_path, fname='lcmv-vol')
+
+src = mne.read_forward_solution(fname_fwd)['src']
+stc = mne.read_source_estimate('lcmv-vol-vl.stc')
+stc_rh = mne.read_source_estimate('lcmv-surf-rh.stc')
+
+###############################################################################
+# Morph VolSourceEstimate
+
+# initialize morpher for VolSourceEstimate (src=src)
+morpher = SourceMorph('sample',
+                      'fsaverage',
+                      src=src,
+                      subjects_dir=subjects_dir,
+                      grid_spacing=(5., 5., 5.))
+
+# save morph
+morpher.save('vol')
+
+# load morph by initializing with file name
+morpher_loaded = SourceMorph(None, None, fname='vol-morph.h5')
+
+# morph data
+stc_morphed = morpher_loaded(stc)
+
+###############################################################################
+# Morph SourceEstimate
+
+fs_vertices = [np.arange(10242)] * 2
+
+# initialize morpher for SourceEstimate (src='surf')
+morpher_rh_surf = SourceMorph('sample', 'fsaverage',
+                              data_from=stc_rh.vertices,
+                              data_to=fs_vertices,
+                              subjects_dir=subjects_dir,
+                              src='surf')
+
+# save morph
+morpher_rh_surf.save('surf_rh')
+
+# load morph by initializing with file name
+morpher_rh_surf_loaded = SourceMorph(None, None, None,
+                                     fname='surf_rh-morph.h5')
```

Comment:
If I'm a user learning MNE, I will wonder what all these `None`s are ...

Reply:
the function is already completely different - no None s confusion anymore ;)